src/styles/main.scss

```diff
@@ -1,3 +1,29 @@
+@import "utils/variables";
+@import "utils/functions";
+@import "reset";
+
 body {
   margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+  overflow-x: hidden;
 }
+
+.header {
+  width: calc(100vw - 2 * #{$horizontal-padding});
```

Comment:
Don't use `vw`, if you were to set 100vw on layouts that have vertical scroll this would cause an overflow adding a horizontal scroll

Reply:
I replaced `vw` to `100%`.